core/src/main/java/io/substrait/extended/expression/ExtendedExpressionProtoConverter.java

```diff
@@ -0,0 +1,50 @@
+package io.substrait.extended.expression;
+
+import io.substrait.expression.Expression;
+import io.substrait.expression.proto.ExpressionProtoConverter;
+import io.substrait.extension.ExtensionCollector;
+import io.substrait.proto.ExpressionReference;
+import io.substrait.proto.ExtendedExpression;
+import io.substrait.type.proto.TypeProtoConverter;
+
+/**
+ * Converts from {@link io.substrait.extended.expression.ExtendedExpression} to {@link
+ * ExtendedExpression}
+ */
+public class ExtendedExpressionProtoConverter {
+  public ExtendedExpression toProto(
+      io.substrait.extended.expression.ExtendedExpression extendedExpression) {
+
+    ExtendedExpression.Builder extendedExpressionBuilder = ExtendedExpression.newBuilder();
```

Comment:
```suggestion
    ExtendedExpression.Builder builder = ExtendedExpression.newBuilder();
```
nit: this is just for readability